Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -368,8 +371,6 @@ namespace AzToolsFramework
             delete m_containerAddButton;
         }
 
-        this->unsetCursor();
```

Comment:
I think we need to leave this call to `unsetCursor` because the property row widgets don't get deleted, but instead get returned to a pool of widgets to be re-used.

Reply:
Leaving that call, at least where it was, is what was preventing the cursor from changing on the two buttons (in addition to the lack of setting the cursor). If it is necessary to have `unsetCursor` called then I can look into moving it above the initialization of the widgets. I tried moving it below all widget initialization and it prevents the cursor from changing over any of the buttons.

Reply:
Yeah try moving it up to the top of the `Initialize` method, below this block:

```
m_initialized = true;
m_hadChildren = false;
m_parentRow = pParent;
m_isMultiSizeContainer = false;
m_isSelected = false;
...
```

Reply:
@cgalvan That seemed to do the trick. Committed and pushed the change.

Reply:
Great, thanks! I'll kick off an AR run now.